molvis/molvisgl/src/processors/molecularrenderer.cpp

```diff
@@ -380,16 +444,56 @@ std::shared_ptr<Mesh> MolecularRenderer::createMesh(const molvis::MolecularStruc
     return mesh;
 }
 
+void MolecularRenderer::updateBrushing(Mesh& mesh, size_t offset) {
```

Comment:
Maybe this function could be shared?

Reply:
Resolved. Moved to `util/molvisglutils.h`